### PR TITLE
LPD-21105 Remove FF

### PR DESCRIPTION
**Pull request by sandrodw3**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:relevant

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ViewportSizeSelector.tsx [hunk 1]
```diff
@@ -72,30 +68,28 @@ export default function ViewportSizeSelector({
...
-					selectedItem={selectedSize}
-					selectedKey={selectedSize}
-				>
-					{({icon, label, sizeId}) => (
-						<Option key={sizeId} textValue={label}>
-							<span className="inline-item inline-item-before ml-1 mr-3">
-								<ClayIcon symbol={icon} />
-							</span>
+			<Picker
+				UNSAFE_menuClassName="cadmin"
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ViewportSizeSelector.tsx:72]: 😱 UNSAFE 😱

Comment (sandrodw3) [reply modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/ViewportSizeSelector.tsx:72]: Who's this guy?

**Comment by sandrodw3** [comment]
ci:test:sf

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:forward:force

**Pull request closed without merging**